test/f90_correct/inc/use00.mk

```diff
@@ -0,0 +1,20 @@
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+########## Make rule for test use00  ########
+
+FFLAGS+= -Mallocatable=03
```

Comment:
Why is this needed for the test?

Reply:
Sorry, copy & paste from another test. Removed now :)